macro-analytics/appsscript.json

```diff
@@ -0,0 +1,12 @@
+{
+    "timeZone": "America/Los_Angeles",
```

Comment:
Nit: indentation seems off, it looks like it starts using 4 spaces here but the rest of the contents use 2 spaces so it doesn't close at the same level by the end. I'd say to remove 2 spaces so everything is indented 2 spaces.

Reply:
good call, added!